bucket_D9/opencvs/specification

```diff
@@ -0,0 +1,33 @@
+DEF[PORTVERSION]=	20170828
+# ----------------------------------------------------------------------------
```

Comment:
So I was suggesting something like this:
DEF[RELEASE_TAG]= 20180929

and then on SITES{main], change it to 
```
SITES[main]=		GITHUB/OrangeGrayCyan:cvs:${RELEASE_TAG}
```
I think that would make maintenance easier.
If you need to change the tag, you'd add the line:
```
REVISION=      1
```
You can even make the REVISION value another variable so you can keep all the definitions together on the first 3 lines.